docroot/modules/custom/mass_unpublish_reminders/templates/mass-reminder-mail-template.html.twig

```diff
@@ -0,0 +1,8 @@
+<p>A page or alert that you authored has an unpublish date that will arrive soon.  At that time, the page or alert will be unpublished.</p>
+<p>Page: {{ page_url }}</p>
+<p>Unpublish date: {{ unpublish_date }}</p>
+<p>If you want to keep the page or alert, please review it, check it's performance, and update it if necessary.  You can then update the unpublish date to a new date.</p>
```

Comment:
```suggestion
<p>If you want to keep the page or alert, please review it, check its performance, and update it if necessary.  You can then update the unpublish date to a new date.</p>
```

Reply:
I don't think we should touch this. this is an approved version with changes made by Joe.